Project.toml

```diff
@@ -5,9 +5,17 @@ version = "0.12.0-dev"
 [deps]
 FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StyledStrings = "f489334b-da3d-4c2e-b8f0-e476e12c162b"
```

Comment:
Should it be deleted from `[deps]` and thus only in `[weakdeps]`?

Reply:
It's in both for Julia < 1.9 compatibility. As per https://pkgdocs.julialang.org/dev/creating-packages/:

> Make sure that the package is both in the [deps] and [weakdeps] section. Newer Julia versions will ignore dependencies in [deps] that are also in [weakdeps].

Though I do see I forgot to do the other half of that :sweat_smile:, one sec.

Reply:
Ah, if this is done you'll want to hold off making a new release until StyledStrings is registered in General (I have a 1.0-compatible version lined up).

Alternatively, we can leave this as a weakdep-only for now, and then make it plain dep in Julia < 1.9 after StyledStrings is added to General.

Reply:
I'll change the PR to "weakdep only" for now, since that's the more conservative option and is the easiest to think about.